BlitzBoard/Features/FixturesScreenViewModel.swift

```diff
@@ -0,0 +1,52 @@
+//  FixturesScreenViewModel.swift
+//  BlitzBoard
+//
+//  Created by Kiasha Rangasamy on 2024/05/19.
+//
+
+import Foundation
+
+class FixturesScreenViewModel {
+    
+    // MARK: Variables
+    
+    var fixtures: [FixturesModel]
+    private let repository: FixturesRepositoryType
+    private weak var delegate: ViewModelDelegate?
+    
+    init(repository: FixturesRepositoryType, delegate: ViewModelDelegate) {
+        self.repository = repository
+        self.delegate = delegate
+        self.fixtures = []
+        
+    }
+    
+    // MARK: Computed Properties
+    
+    var fetchNumberOfGames: Int {
```

Comment:
```suggestion
    var numberOfGames: Int {
```
No need to use the word fetch for just extracting variables